packages/footer/hooks/useTrackedOnClick.tsx

```diff
@@ -4,18 +4,25 @@ import { TrackingContext } from './TrackingContext';
 type UseFooterLinkTrackingProps<T> = {
    href?: string;
    onClick?: React.MouseEventHandler<T>;
+   clickName?: string;
+   isStoreLink?: boolean;
 };
 
 export function useTrackedOnClick<T>({
    href,
    onClick,
+   clickName,
+   isStoreLink,
 }: UseFooterLinkTrackingProps<T>) {
+   const LinkType = isStoreLink ? 'Stores' : 'iFixit';
```

Comment:
(minor) We should probably name this `linkType` since it's a regular variable.

Reply:
Good catch.

Reply:
Fixed in [1f47c68](https://github.com/iFixit/react-commerce/pull/1069/commits/1f47c689fb1fe0a001515ce91f1e717afe0927ee)